msu/systems/mod_settings/mod_settings_system.nut

```diff
@@ -133,14 +133,14 @@
 		this.Screen.updateSettingInJS( _modID, _settingID, _value );
 	}
 
-	function callPanelsFunction( _function, ... )
+	function callPanelsFunction( _function, _argsArray )
 	{
-		vargv.insert(0, null);
+		_argsArray.insert(0, null);
 
 		foreach (panel in this.Panels)
 		{
-			vargv[0] = panel;
-			panel[_function].acall(vargv);
+			_argsArray[0] = panel;
+			panel[_function].acall(_argsArray);
 		}
 	}
```

Comment:
Why are we changing this one?

Reply:
Because it was arbitrarily different from the skill_container callSkillsFunction and since it wasn't used at all before it doesn't actually matter.